LabDrivers/Tool.py

```diff
@@ -717,21 +765,49 @@ def __del__(self):
         logging.info("InstrumentHub deleted")
 
     def change_interface(self, intf):
+        global INTF_GPIB
```

Comment:
Why do you need a global variable here? You can access the INTF_GPIB value in the config.txt file. You can also update it there if needed, there are functions in `LabTools.IO`